### DISCUSSION TITLE
HTML standalone web3modal for auth for Algorand

### DISCUSSION THREAD

**Discussion by 1m1-github** [WalletConnect/walletconnect-monorepo#2388] [Developer Support]
i am running this example: https://github.com/WalletConnect/web3modal-examples/tree/main/html-auth

i want to connect to the Algorand chain

i tried adding chainId:
`const data = await web3Modal.signIn({ chainId: "4160", statement: "Connect to Web3Modal" });`
but Pera (an Algorand walletconnect wallet) does not find the url valid

i also saw https://docs.walletconnect.com/2.0/web3modal/advanced/standalone/auth/installation
which is a little differently coded from the example at the top

**Comment by xzilja** [comment]
That example is for new auth libraries (html & react) that are being documented this week. They will simplify working with WalletConnect's auth client and web3modal, as you noticed. chainId needs to be in [CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md) compliant format i.e. `namespace:chainId`, which for ethereum would be `eip155:1`

From the top of my head I don't know namespace for Algorand, so you'd need to find that. Also, I'm not sure if Pera wallet supports our auth sdk yet.

**Comment by 1m1-github** [comment]
do you know of any Algorand wallet that would support it?

**Reply by finessevanes** [reply]
Any Algorand wallet that currently supports Auth?

**Reply by drichar** [reply]
The Algorand namespace was just added today: https://github.com/WalletConnect/walletconnect-monorepo/issues/2538

Keep an eye out for wallets with 2.0 support in the coming weeks.
